pkg/storaged/client.js

```diff
@@ -344,11 +345,18 @@ export async function btrfs_poll() {
     btrfs_update(data);
 }
 
+let btrfs_monitor_channel = null;
+
 function btrfs_start_monitor() {
     if (!client.superuser.allowed || !client.features.btrfs) {
         return;
     }
 
+    if (btrfs_monitor_channel)
+        return;
+
+    console.log("STARTING");
```

Comment:
debugging leftover (more instances below)

Reply:
Yeah, I wanted to see them in the logs...